src/appleseed/renderer/modeling/camera/camera.h

```diff
@@ -194,6 +207,15 @@ class APPLESEED_DLLSYMBOL Camera
 };
 
 
+//
+// Shutter curves. Works in Camera::map_to_shutter_curve().
+//
+
+// Integrate and inverse opening/closing curve y = a * x + b to transform samples.
+namespace
```

Comment:
It doesn't make much sense to forward-declare a free function from an anonymous namespace since it can't be used anyway (being in an anonymous namespace).

You can simply remove the whole declaration here.